src/Compilers/Core/Portable/PEWriter/BlobWriter.cs

```diff
@@ -628,6 +628,23 @@ internal void WriteCompressedUInt(uint val)
             }
         }
 
+        internal static int GetCompressedIntegerSize(int value)
```

Comment:
Look at the previous method ;-) Any suggestions? I'm not sure how to document these other than "see ECMA spec for compressed integer definition". Naming the constants MaxSingleByteValue, MaxTwoByteValue, etc. doesn't seem to add too much value since that's exactly what is already expressed in the code.